.github/workflows/test.yaml

```
@@ -80,3 +80,17 @@ jobs:
+  
+  e2e-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Test E2E satellite
+        uses: dagger/dagger-for-github@v7
+        with:
+          version: ${{ steps.dagger_version.outputs.version }}
```

actionlint (1.7.7)

property "dagger_version" is not defined in object type {}

(expression)